src/rinoh/backend/pdf/__init__.py

```diff
@@ -421,11 +421,11 @@ def _convert_to_png(self, filename_or_file):
 
 def _code_to_char(code):
     if code < 32 or code > 127:
-        char = '\{:03o}'.format(code)
+        char = r'\{:03o}'.format(code)
     else:
         char = chr(code)
-        if char in ('\\', '(', ')'):
-            char = '\\' + char
+        if char in (r'\\', r'(', r')'):
```

Comment:
`r'\\'` is a two-character string. `char` is a single-character string, it never can be equal to `r'\\'`.

I think the existing code (without `r`) is correct.